script/tool/lib/src/lint_podspecs_command.dart

```diff
@@ -103,19 +90,19 @@ class LintPodspecsCommand extends PluginCommand {
     final String podspecPath = podspec.path;
 
     final String podspecBasename = p.basename(podspecPath);
-    _print('Linting $podspecBasename');
+    print('Linting $podspecBasename');
 
     // Lint plugin as framework (use_frameworks!).
     final ProcessResult frameworkResult =
         await _runPodLint(podspecPath, libraryLint: true);
-    _print(frameworkResult.stdout);
-    _print(frameworkResult.stderr);
+    print(frameworkResult.stdout);
+    print(frameworkResult.stderr);
```

Comment:
nit: This seems like it would be better if we were streaming directly instead of buffering it up then spitting it out.  Completely out of scope for this PR, just something to keep an eye on.